book/src/themes.md

```diff
@@ -70,6 +70,7 @@ over it and is merged into the default palette.
 
 | Color Name      |
 | ---             |
+| `reset`         |
```

Comment:
Perhaps the documentation could use an update. 
> The default palette uses the terminal's default 16 colors
I might send a PR at some point making some clarifications to this documentation chapter in general.